style.css

```diff
@@ -0,0 +1,536 @@
+  background-size: cover;
+  color: red;
+  -webkit-text-fill-color: transparent;
+  -webkit-background-clip: text;
+  -webkit-box-shadow: inset 0 0 6px rgba(0, 0, 0, 0.1);
+  box-shadow: inset 0 0 6px rgba(0, 0, 0, 0.1);
+  background-color: #000;
+}
+
+.place {
+  margin: 0;
+  font-size: 18px;
+  font-weight: 500;
+  font-family: 'Lato', sans-serif;
+  margin-top: 4px;
+}
+
+.time {
+  margin: 2px;
+  font-weight: 800;
+  font-family: 'Lato', sans-serif;
+  font-size: 26px;
+}
+
+.banner {
+  background-image: url(./images/background.svg);
+  background-repeat: no-repeat;
+  background-size: 100%;
+  padding-bottom: 100px;
+  padding-top: 100px;
+  padding-left: 10%;
+}
+
+.log {
+  width: 100px;
+  height: 65px;
+}
+
+.detail {
+  font-family: 'Lato', sans-serif;
+  margin-left: 20px;
+  padding: 5px;
+  box-shadow: inset 0 0 6px rgba(0, 0, 0, 0.1);
+}
+
+.programs {
+  padding: 20px 20px 40px 20px;
+  padding-top: 3%;
+  padding-bottom: 3%;
+  background-color: hsl(240, 5%, 18%);
+  background-image: url(./images/cross.svg);
+  background-size: 32px;
+}
+
+.title {
+  height: 10vh;
+  justify-content: center;
+  color: #fff;
+  text-align: center;
+}
+
+.grid-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  gap: 5px;
+  padding: 20px;
+  padding-left: 100px;
+  padding-right: 100px;
+}
+
+.grid1 {
+  width: auto;
+  height: auto;
+  background-color: rgba(255, 255, 255, 0.122);
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  justify-content: center;
+  text-align: center;
+  padding: 10px;
+}
+
+.selected {
+  color: orangered !important;
+}
+
+button {
+  display: none;
+}
+
+.featured-speakers-list {
+  padding: 5%;
+  height: auto;
+
+  /* display: flex; */
+
+  /* height: 80vh; */
+}
+
+.bigDiv {
+  display: flex;
+
+  /* flex-direction: row; */
+  flex-wrap: wrap;
+  gap: 20px;
+  padding-top: 69px;
+  justify-content: center;
+}
+
+.context h2 {
+  text-align: center;
+}
+
+.contain {
+  width: 42vw;
+  display: flex;
+  font-family: 'Lato', sans-serif;
+}
+
+.bg-text h2 {
+  /* padding: 15px; */
+  margin: 10px;
+  font-size: 20px;
+  font-family: 'Lato', sans-serif;
+  text-align: start;
+}
+
+.bg-text h3 {
+  color: red;
+  margin: 10px;
+  font-size: 16px;
+  font-style: italic;
+  font-family: 'Lato', sans-serif;
+  text-align: start;
+}
+
+.bg-text p {
+  font-size: 14px;
+  margin: 10px;
+  font-family: 'Lato', sans-serif;
+  font-weight: 300;
+  text-align: start;
+}
+
+.gray-dash {
+  background-color: black;
+  width: 30px;
+  height: 2px;
+  margin: 10px;
+}
+
+.trans-bg {
+  width: 10vw;
+  height: 18vh;
+  background-image: url(./images/sec.jpg);
+  background-position: top left;
+  background-size: 38%;
+  background-repeat: no-repeat;
+}
+
+.trans-bg img {
+  width: 7vw;
+  height: 15vh;
+  padding: 5px;
+  margin-top: 5px;
+  margin-left: 10px;
+}
+
+.partners {
+  background-color: #424040;
+  height: 20vh;
+}
```

Comment:
- [ ] In this project, we are supposed to use exact same colors, for text, backgrounds, borders, etc, from the original design but currently, you are using different colors on your page, I think it would be better to replicate the original design color. ✨

   _Remember, If you're working on a project for a client, they likely have a specific color palette associated with their brand. By using the same colors as given in the design, you're helping to reinforce their brand identity._

  Your may try to follow the basic color swatch: ```#272A31, #EC5242, #D3D3D3``` and also you can use the[ color picker for chrome](https://chrome.google.com/webstore/detail/color-picker-for-chrome/clldacgmdnnanihiibdgemajcfkmfhia?gclid=CjwKCAiA0JKfBhBIEiwAPhZXDwDdoqMT1lw0W4Q_rLSWRtZRxJ_kKw-SldF0fYgvxtkTzEfmXYhAbRoCR1YQAvD_BwE) extension to find color code on any page 😉😉
  
**Figma design** | **Your design**
| --- | --- |
![image](https://user-images.githubusercontent.com/102757126/217866548-f9d15646-f158-4431-a627-faa14a0b66b6.png) | ![image](https://user-images.githubusercontent.com/102757126/217866806-6580b4c5-8423-40eb-9688-55407ec65f80.png)

- [ ] ***Kindly try to check all the colors on your page to that of the original design 🙏***

Reply:
Fixed.